package.json

```diff
@@ -24,5 +24,6 @@
     "msw": "^0.39.2"
   },
   "author": "Brian Gantzler",
```

Comment:
You can change this name to something more generic if you wish, or even just MirageJS

Reply:
You're the one who wrote it originally, seems fair to keep you as the author.